src/components/Modals/FiatRamps/fiatRampProviders/mtpelerin.ts

```diff
@@ -36,7 +36,12 @@ export async function getMtPelerinAssets(): Promise<AssetId[]> {
   ).filter(isSome)
 }
 
-export const createMtPelerinUrl = (action: FiatRampAction, assetId: AssetId): string => {
+export const createMtPelerinUrl = (
+  action: FiatRampAction,
+  assetId: AssetId,
+  _address: string,
```

Comment:
Let's not get burned by arity madness ?

<img width="427" alt="image" src="https://user-images.githubusercontent.com/17035424/201428468-911e83a3-9f84-4d69-8728-9aacb0287b26.png">